src/Certificate/Quiz.jsx

```diff
@@ -1,14 +1,15 @@
 import React, { Component } from 'react';
-import { Form, FormError, RadioGroup, Radio } from 'react-form';
+import { Form, RadioGroup, Radio } from 'react-form';
```

Comment:
why is this removed?

Reply:
router-form v2 doesn't have FormError it seems ... :-(
https://react-form.js.org/#/
https://github.com/react-tools/react-form

Reply:
oh sad :/